src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -157,7 +157,7 @@ internal FileData CreateFileData(FileDataVersionOne v1FileData)
             {
                 fileData = new FileData
                 {
-                    Hashes = BuildHashesDictionary(v1FileData.Hashes),
+                    Hashes = v1FileData.Hashes?.Select(CreateHash).ToDictionary(p => p.Key, p => p.Value),
```

Comment:
>CreateHash [](start = 55, length = 10)

Because `CreateHash` does the `ToLowerInvariant` call that `BuildHashesDictionary` lacked.

Reply:
No, I see, it's because it handles values that are _not_ in `Utilities.AlgorithmKindNameMap` -- which "md5" is not.

---
In reply to: [254911070](https://github.com/Microsoft/sarif-sdk/pull/1266#discussion_r254911070) [](ancestors = 254911070)